dev/VSIX/Extension/ProjectReunion.Extension.csproj

```diff
@@ -220,6 +223,20 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <IncludeOutputGroupsInVSIX>TemplateProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
     </ProjectReference>
+      <ProjectReference Include="..\ProjectTemplates\Desktop\CppWinRT\SingleProjectPackagedApp\WinUI.Desktop.CppWinRT.SingleProjectPackagedApp.csproj">
```

Comment:
indentation.  also, shouldn't these both have a  Condition="'$(EnableExperimentalVSIXFeatures)'=='true'" ?

Reply:
The condition is on the enclosing ItemGroup.